packages/kolibri-common/components/Cards/AccessibleFolderCard.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+
+  
+  <KCard
+    :to="to"
+    :headingLevel="headingLevel"
+    layout="horizontal"
+    thumbnailDisplay="large"
+    :title="contentNode.title"
+    :thumbnailSrc="thumbnailSrc"
+    thumbnailScaleType="centerInside"
+    thumbnailAlign="right"
+    :style="{ height: '172px', margin: '16px 0 16px 0' }"
+  >
+    <template #thumbnailPlaceholder>
+      <div class="default-folder-icon" >
+        <KIcon
+          icon="topic"
+          :color="$themePalette.grey.v_600"
+          class=""
+        />
+      </div>
+    </template>
+
+    <template #belowTitle>
+      <div
+        class="header-bar"
+        :style="headerStyles "
+      >
+        <KIcon
+          icon="topic"
+          :color="$themePalette.grey.v_800"
+          class="folder-header-bar"
+        />
+        <p
+          class="folder-header-text"
+          :style="{ color: $themePalette.grey.v_600 }"
+        >
+          {{ coreString('folder') }}
+        </p>
+      </div>
+    </template>
+
+  </KCard>
+
+</template>
+
+
+<script>
+
+  import { validateLinkObject } from 'kolibri.utils.validators';
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+
+  export default {
+    name: 'AccessibleFolderCard',
+    mixins: [commonCoreStrings],
+    props: {
+      to: {
+        type: Object,
+        required: true,
+        validator: validateLinkObject,
+      },
+      contentNode: {
+        type: Object,
+        required: true,
+      },
+      headingLevel: {
+        type: Number,
+        required:true
+      },
+      thumbnailSrc: {
+        type: String,
+        default: null,
+      },
```

Comment:
Non-blocking question: 

Is this prop here so that we can override the content's thumbnail value? Is there an expected case of using anything besides `contentNode.thumbnail` for the thumbnail here?

Reply:
Yes , the prop thumbnailSrc is to override the value of the KImg in the KCard. No I think the chances of using anything besides contentNode.thumbnail are very minimal